src/Mono.Data.Sqlite/Mono.Data.Sqlite.csproj

```diff
@@ -11,15 +11,13 @@
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <AndroidResgenClass>Resource</AndroidResgenClass>
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
-    <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
     <AssemblyName>Mono.Data.Sqlite</AssemblyName>
-    <SignAssembly>True</SignAssembly>
-    <AndroidUseSharedRuntime>False</AndroidUseSharedRuntime>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\product.snk</AssemblyOriginatorKeyFile>
```

Comment:
This is a different signing key than what `Mono.Data.Sqlite.dll` is currently built with, so I believe this would break existing 3rd party assemblies.